### PR TITLE
[3aa410b] fix property editor translation and add make unique translation

### PR DESCRIPTION
**Pull request by sersoong**
Please pay attention to this matter
capture

### CODE REVIEW

**Comment by groud** [comment]
Please use the dedicated platform to update the translation:
https://hosted.weblate.org/projects/godot-engine/godot/

Also, I don't get the changes in the README.md file.

I'm ok with the missing TTR though.

**Comment by groud** [comment]
@akien-mga sorry for the requested review, I missclicked :/

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Official binaries for the Godot editor and the export templates can be found
 
 #### Compiling from source
 
-[See the official docs](http://docs.godotengine.org/en/latest/development/compiling/)
+[See the official docs](http://docs.godotengine.org/en/latest/reference/_compiling.html)
```
Comment (Noshyaar) [inline README.md]: Please do not change this.

File: README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ on Freenode.
 The official documentation is hosted on [ReadTheDocs](http://docs.godotengine.org).
 It is maintained by the Godot community in its own [GitHub repository](https://github.com/godotengine/godot-docs).
 
-The [class reference](http://docs.godotengine.org/en/latest/classes/)
+The [class reference](http://docs.godotengine.org/en/latest/classes/_classes.html)
```
Comment (Noshyaar) [inline README.md]: Please do not change this.

File: README.md [hunk 3]
```diff
@@ -63,4 +63,4 @@ such as text and video tutorials, demos, etc. Consult the [community channels](h
 for more info.
 
 [![Build Status](https://travis-ci.org/godotengine/godot.svg?branch=master)](https://travis-ci.org/godotengine/godot)
-[![Code Triagers Badge](https://www.codetriage.com/godotengine/godot/badges/users.svg)](https://www.codetriage.com/godotengine/godot)
+[![Code Triagers Badge](http://www.codetriage.com/godotengine/godot/badges/users.svg)](http://www.codetriage.com/godotengine/godot)
```
Comment (Noshyaar) [inline README.md]: Please do not change this.

File: editor/translations/zh_CN.po [hunk 4]
```diff
@@ -1362,11 +1362,15 @@ msgstr "拷贝资源"
 
 #: editor/editor_node.cpp
 msgid "Make Built-In"
-msgstr "使之内置"
+msgstr "转为内置"
```
Comment (Noshyaar) [inline editor/translations/zh_CN.po]: We use Weblate to manage translation. Do not change this file.

**Review by ghost — CHANGES_REQUESTED** [review]
- Please fix as commented.
- Please rebase. [Guide](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase)

**Pull request closed without merging**